varinfo/generate_umm_var.py

```diff
@@ -69,7 +72,7 @@ def generate_collection_umm_var(collection_concept_id: str,
         # error messages returned from CMR.
         publication_response = publish_all_umm_var(collection_concept_id,
                                                    all_umm_var_records,
-                                                   auth_header, cmr_env)
+                                                   launchpad_token, cmr_env)
```

Comment:
After all my comments, I'm expecting: "but... if someone gives us an auth header with an EDL bearer token, the record ingest will fail". Totally true. But in that case, I think we can do one of two things:

1) Rely on the CMR API to reject the request and tell us why.
2) We could wrap this call with a check that the input auth header isn't a EDL bearer token header.

Reply:
Haha I was going to say that but also realized that yeah that's more the CMR API's job to tell us why it failed. Failed ingest due to Authorization errors are usually clear.